Manta build failure

Seems there is a problem with the manta bioconda build, see discussion in https://github.com/bioconda/bioconda-recipes/issues/53419 (relates to update in https://github.com/bioconda/bioconda-recipes/pull/52517)

> configManta.py --bam final.phased.cram --referenceFasta /scratch/user/ajd/Serina/TellSeq2/ref_genomes/S_tuberosum_v6.1/Stuberosum_686_v6.1.fa --callRegions primary.bed.gz --runDir manta_tmp &> manta.log && python2 manta_tmp/runWorkflow.py --jobs 20 &>> manta.log && convertInversion.py $(which samtools) /scratch/user/ajd/Serina/TellSeq2/ref_genomes/S_tuberosum_v6.1/Stuberosum_686_v6.1.fa manta_tmp/results/variants/diploidSV.vcf.gz 2> manta.log |  bgzip -c > manta_tmp/results/variants/diploidSV.convertInversion.vcf.gz && cp manta_tmp/results/variants/diploidSV.convertInversion.vcf.gz final.manta_sv_calls.vcf.gz && cp manta_tmp/results/variants/candidateSmallIndels.vcf.gz final.manta_cand_small.vcf.gz && cp manta_tmp/results/variants/candidateSmallIndels.vcf.gz.tbi final.manta_cand_small.vcf.gz.tbi && cp manta_tmp/results/variants/candidateSV.vcf.gz final.manta_cand_sv.vcf.gz && cp manta_tmp/results/variants/candidateSV.vcf.gz.tbi final.manta_cand_sv.vcf.gz.tbi
> Activating conda environment: /scratch/user/ajd/Serina/TellSeq2/Outputs_V3/KTZ_1_V5/Outputs/.snakemake/conda/6437933bd16a31cc5b317d5462f11a75
> [Wed Jan 29 07:42:21 2025]
> Error in rule manta:
>     jobid: 551
>     output: final.manta_sv_calls.vcf.gz, final.manta_cand_small.vcf.gz, final.manta_cand_small.vcf.gz.tbi, final.manta_cand_sv.vcf.gz, final.manta_cand_sv.vcf.gz.tbi, manta_tmp, manta_tmp/runWorkflow.py
>     log: manta.log (check log file(s) for error message)
>     conda-env: /scratch/user/ajd/Serina/TellSeq2/Outputs_V3/KTZ_1_V5/Outputs/.snakemake/conda/6437933bd16a31cc5b317d5462f11a75
>     shell:
>         configManta.py --bam final.phased.cram --referenceFasta /scratch/user/ajd/Serina/TellSeq2/ref_genomes/S_tuberosum_v6.1/Stuberosum_686_v6.1.fa --callRegions primary.bed.gz --runDir manta_tmp &> manta.log && python2 manta_tmp/runWorkflow.py --jobs 20 &>> manta.log && convertInversion.py $(which samtools) /scratch/user/ajd/Serina/TellSeq2/ref_genomes/S_tuberosum_v6.1/Stuberosum_686_v6.1.fa manta_tmp/results/variants/diploidSV.vcf.gz 2> manta.log |  bgzip -c > manta_tmp/results/variants/diploidSV.convertInversion.vcf.gz && cp manta_tmp/results/variants/diploidSV.convertInversion.vcf.gz final.manta_sv_calls.vcf.gz && cp manta_tmp/results/variants/candidateSmallIndels.vcf.gz final.manta_cand_small.vcf.gz && cp manta_tmp/results/variants/candidateSmallIndels.vcf.gz.tbi final.manta_cand_small.vcf.gz.tbi && cp manta_tmp/results/variants/candidateSV.vcf.gz final.manta_cand_sv.vcf.gz && cp manta_tmp/results/variants/candidateSV.vcf.gz.tbi final.manta_cand_sv.vcf.gz.tbi
>         (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)
> ```
> 
> Looking specifically within the manta.log file, there was a repeated line that mentioned the following issue with the getAlignmentStats_generateStats_000 task:
> 
> ```[2025-01-29T13:39:03.556938Z] [c315.cluster] [154838_1] [TaskRunner:getAlignmentStats_generateStats_000] [WARNING] Task: 'getAlignmentStats_generateStats_000' failed but qualifies for retry. Total task failures (including this one): 1. Task command: '/scratch/user/ajd/Serina/TellSeq2/Outputs_V3/KTZ_1_V5/Outputs/.snakemake/conda/6437933bd16a31cc5b317d5462f11a75/libexec/GetAlignmentStats --ref /scratch/user/ajd/Serina/TellSeq2/ref_genomes/S_tuberosum_v6.1/Stuberosum_686_v6.1.fa --output-file /scratch/user/ajd/Serina/TellSeq2/Outputs_V3/KTZ_1_V5/Outputs/manta_tmp/workspace/alignmentStats.xml.tmpdir/alignmentStats.xml.000.xml --align-file /scratch/user/ajd/Serina/TellSeq2/Outputs_V3/KTZ_1_V5/Outputs/final.phased.cram'```
> 
> The final line of manta.log contained the following:
> `
> [2025-01-29T13:42:21.463944Z] [c315.cluster] [154838_1] [WorkflowRunner] [ERROR] [2025-01-29T13:42:05.043215Z] [c315.cluster] [154838_1] [getChromDepth_sample000_estimateChromDepth_245_scaffold_355_to_261_scaffold_373] /scratch/user/ajd/Serina/TellSeq2/Outputs_V3/KTZ_1_V5/Outputs/.snakemake/conda/6437933bd16a31cc5b317d5462f11a75/libexec/GetChromDepth: error while loading shared libraries: libboost_filesystem.so.1.85.0: cannot open shared object file: No such file or directory`
> 
> I'm attaching both log files in case these are helpful at all. I am happy to provide any other details that may be of assistance. Thanks again for your help!
> 
> [tellseq_ktz1.txt](https://github.com/user-attachments/files/18589334/tellseq_ktz1.txt)
> [manta.log](https://github.com/user-attachments/files/18589333/manta.log) 

 _Originally posted by @ajdesalvio in [#85](https://github.com/AfshinLab/BLR/issues/85#issuecomment-2621883326)_

--- a/src/blr/envs/manta.yml
+++ b/src/blr/envs/manta.yml
@@ -4,6 +4,6 @@ channels:
   - bioconda
   - nodefaults
 dependencies:
-  - bioconda::manta=1.6.0
+  - bioconda::manta[version=1.6.0, build=*_3]
   - bioconda::samtools=1.12
-  - conda-forge::python<3
+  - conda-forge::python=2.7*

